qemu/tests/cfg/virtio_fs_localfs_migration.cfg

```diff
@@ -0,0 +1,122 @@
+- virtio_fs_localfs_migration:
+    no RHEL.6 RHEL.7 RHEL.8.0 RHEL.8.1
+    no Win2008 Win7
+
+    type = virtio_fs_localfs_migration
+    virt_test_type = qemu
+    required_qemu = [9.0.0,)
+    kill_vm = yes
+    start_vm = yes
+
+    filesystems = fs
+    filesystems_migration = targetfs
+    fs_driver = virtio-fs
+    fs_source_type = mount
+    fs_source_dir_fs = /var/tmp/virtio_fs_source
+    fs_source_dir_targetfs = /var/tmp/virtio_fs_target
```

Comment:
Another thing to confirm with @hellohellenmao : 
The migration happened on two nodes and the systemdisk is located in NFS server, is that correct?

Reply:
IMO systemdisk is not a problem as we are doing single node migration, the fs exported is the key

Reply:
Make sense. Then it's okay for single node migration.